docs/guides/hardhat-walkthrough/hardhat-base-setup.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_label: Hardhat basic setup
+sidebar_position: 1
+---
+
+# Setup your Hardhat project
+
+In this article will guide you through the process of seting up an Hardhat installation (using TypeScript), adding the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package, creating a basic LSP7 contract, and deploying it on [LUKSO Testnet](http://docs.lukso.tech/networks/testnet/parameters).
+
+## Create Hardhat project
+
+The first thing to do is to create a new Hardhat project that will use TypeScript:
+
+```bash title="Setup new hardhat project"
+mkdir lukso-app
+cd lukso-app
+npx hardhat # select 'Create a TypeScript project' and use the default value for the rest of the setup
+```
+
+Once finished, you have a working hardhat setup!
+
+## Install packages & setup tools
+
+To work in the best condition possible, we will install libraries that includes tools, helpers and the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package.
+
+### Install dependencies
+
+```js title="Install dependencies"
+npm i -D dotenv prettier
+npm i -s @lukso/lsp-smart-contracts
```

Comment:
maybe this install can be on a specific version so it is reproductible ?

Reply:
done